diff -Naur responses-0.5.0.orig/setup.py responses-0.5.0/setup.py
--- responses-0.5.0.orig/setup.py	2015-10-14 23:39:28.000000000 +0200
+++ responses-0.5.0/setup.py	2016-01-23 08:37:12.103460465 +0100
@@ -30,7 +30,7 @@
 
 tests_require = [
     'pytest',
-    'coverage >= 3.7.1, < 4.0.0',
+    'coverage >= 3.7.1, < 5.0.0',
     'pytest-cov',
     'flake8',
 ]
diff -Naur responses-0.5.0.orig/.travis.yml responses-0.5.0/.travis.yml
--- responses-0.5.0.orig/.travis.yml	2015-10-14 23:39:28.000000000 +0200
+++ responses-0.5.0/.travis.yml	2016-01-23 08:36:44.206161468 +0100
@@ -3,7 +3,6 @@
 python:
   - "2.6"
   - "2.7"
-  - "3.2"
   - "3.3"
   - "3.4"
 cache:
